app/containers/Network/index.js

```diff
@@ -255,9 +255,19 @@ const Network = () => {
         />
         <Route
           exact
-          path={`${path}/access-points/:id`}
+          path={`${path}/access-points/:id/:tab`}
           render={props => <AccessPointDetails locations={locationsTree} {...props} />}
         />
+        <Route
+          exact
+          path={`${path}/access-points/:id`}
+          render={props => (
+            <Redirect
+              to={{ pathname: `${path}/access-points/${props?.match?.params?.id}/general` }}
+            />
+          )}
+        />
+
```

Comment:
no. look at the bottom of this component to see how i did it for the Network tab